extension/encoding/googlecloudlogentryencodingextension/testdata/apploadbalancer/regional_external_application_load_balancer_expected.yaml

```diff
@@ -0,0 +1,122 @@
+resourceLogs:
+  - resource:
+      attributes:
+        - key: gcp.project
+          value:
+            stringValue: elastic-observability
+        - key: cloud.resource_id
+          value:
+            stringValue: loadbalancing.googleapis.com%2Fexternal_regional_requests
+        - key: gcp.resource_type
+          value:
+            stringValue: http_external_regional_lb_rule
+        - key: gcp.label.region
+          value:
+            stringValue: europe-southwest1
```

Comment:
Is this independent of load balancer logs? We should translate this to `cloud.region`, but maybe in another PR?